docs/examples/800_istio.adoc

```diff
@@ -0,0 +1,169 @@
+= Istio Support
+
+== Overview
+
+Coherence Operator 3.1.15 and later supports Istio 1.9.1 and later. You can run the operator and Coherence cluster managed by the operator with Istio sidecar injection enabled.  Coherence caches can be accessed from outside via Coherence*Extend, REST, and other supported Coherence clients.  
+
+== Limitations
```

Comment:
I'm not sure whether I'd jump straight into "limitations" or show examples of it working and different configurations and cover limitations at the end

Reply:
Moved to the end.